Mathlib/Order/Bounds/Extra.lean

```diff
@@ -0,0 +1,52 @@
+/-
+Copyright (c) 2025 Christopher Hoskin. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Christopher Hoskin
+-/
+import Mathlib.Order.Bounds.Basic
+import Mathlib.Data.Set.Prod
+
+/-!
+# Extra results
+
+Further results for `upperBounds` and `lowerBounds`
+
+-/
+
+open Set
+
+universe u v
+
+variable {α : Type u} {β : Type v}
+variable [Preorder α] [Preorder β] {f : α → β}
+
+lemma Monotone.dominated (Hf : Monotone f) {s₁ s₂ : Set α} (h : Dominated (· ≤ ·) s₁ s₂) :
+    Dominated (· ≤ ·) (f '' s₁) (f '' s₂) := fun a ha => by
```

Comment:
With the previous changes, this becomes
```suggestion
lemma Dominated.image (Hf : Monotone f) {s₁ s₂ : Set α} (h : Dominated s₁ s₂) :
    Dominated (f '' s₁) (f '' s₂) := fun a ha => by
```

Reply:
Presumably I then want `(h : Dominated s₁ s₂)` to be the first argument so the dot notation works?

Reply:
And it should be `Dominated.image_of_monotone`?